docs/quick_start/installing.rst

```diff
@@ -114,18 +111,32 @@ You may also install multiple packages in a single command. To install the base
 Installing Fledge downloaded packages
 ######################################
 
-Assuming you have downloaded the packages from the download link given above. Use SSH to login to the system that will host Fledge services. For each Fledge package that you choose to install, type the following command::
+Assuming you have downloaded the packages from the download link given above. Use SSH to login to the system that will host Fledge services. For each Fledge package that you choose to install, type the following command
+
+.. code-block:: console
 
-  sudo apt -y install PackageName
+  sudo apt -y install <filename>
 
-The key packages to install are the Fledge core and the Fledge User Interface::
+.. note::
 
-  sudo DEBIAN_FRONTEND=noninteractive apt -y install ./fledge-1.8.0-armv7l.deb
-  sudo apt -y install ./fledge-gui-1.8.0.deb
+  The downloaded files are named using the package name and the current version of the software. Therefore these names will change over time as new versions are released. At the time of writing the version of the Fledge package is 2.3.0, therefore the package filename is fledge_2.3.0_x86_64.deb on the X86 64bit platform. As a result the filenames shown in the following examples may differ from the names of the files you have downloaded.
+
+The key packages to install are the Fledge core and the Fledge User Interface
+
+.. code-block:: console
 
-You will need to install one of more South plugins to acquire data.  You can either do this now or when you are adding the data source. For example, to install the plugin for the Sense HAT sensor board, type::
+  sudo DEBIAN_FRONTEND=noninteractive apt -y install ./fledge_2.3.0_x86_64.deb
+  sudo apt -y install ./fledge-gui_2.3.0_x86_64.deb
```

Comment:
Fledge gui package  is architecture independent. So its name is `fledge-gui_2.3.0.deb`